### DISCUSSION TITLE
`window-format`  : print row number ?

### DISCUSSION THREAD

**Discussion by fenugrec** [davatorium/rofi#1937] [Q&A] [answered]
Couldn't find info on this after a bit of research; possibly related to #1810  but at a much narrower scope.
Is there a formatter or a way to print each row's number (0 to 9, to match with kb-select-N bindings)  ? 

Thanks !

**Comment by DaveDavenport** [comment, marked as answer]
https://davatorium.github.io/rofi/1.7.5/rofi-theme.5/
* element-index: the widget in the listview's entry keybindable index (1,2,3..0)

test out with:
```bash
$ rofi -show drun -theme-str 'element { children: [ element-index, element-text]; }'
```

**Reply by fenugrec** [reply]
Perfect, thanks. Didn't think of looking at the themes manpage, and unfortunately none of the bundled themes have this feature.
